### ISSUE TITLE
Shall we deprecate Bitnami's based image?

### ISSUE THREAD

**Issue by ernani** [google/moodle-on-gcp#14]
Considering removing this file once Nginx becomes the only setup as it is done via the setup_moodle.sh script with moosh utility in initial setup process.

_Originally posted by @ernani in https://github.com/google/moodle-on-gcp/pull/12#discussion_r1283345770_

**Comment by geronimo794** [comment]
Yes please, even though installing with bitnami base image is easier with the help of helm install. But it will be confusing for newcomer like me to try to install Moodle on gcp if there are 2 options to start with and with different outcomes.

**Comment by ernani** [comment]
We've deprecated it in the docs. We are keeping it there for migration reference only and within 2-3 months we will remove its files completely.

Please do help reviewing them if you can!

Cheers!

**Comment by ernani** [comment]
I have started the process to remove all dependencies from helm and bitnami altogether. This will be a much simpler alternative.
